packages/marketplace/src/components/pages/settings/developer-settings-billing-tab/accounts-information-form/accounts-information-form.tsx

```diff
@@ -32,6 +32,9 @@ const AccountsInformationForm: React.FC<AccountsInformationFormProps> = () => {
   return (
     <Formik initialValues={initialValues} onSubmit={onSubmit}>
       {({ setFieldValue, values }) => {
+        const { reapitAccountsRef, hasReapitAccountsRef } = values
+        const isEnableSaveBtn = hasReapitAccountsRef === 'yes' && reapitAccountsRef.length >= 6
```

Comment:
Should create a constant for max `reapitAccountsRef` field's length and export it so that we can reuse it in `direct-debit-section.tsx`